macos/flutter_web_gl.podspec

```diff
@@ -23,7 +23,7 @@ A new flutter plugin project.
   s.swift_version = '5.0'
   
   s.preserve_paths = 'MetalANGLE.framework'
-  s.xcconfig = { 'OTHER_LDFLAGS' => '-framework MetalANGLE' }
+  s.xcconfig = { 'OTHER_LDFLAGS' => '-framework MetalANGLE', 'HEADER_SEARCH_PATHS' => '"${PODS_TARGET_SRCROOT}/MetalANGLE.framework/Headers"'}
```

Comment:
really? Xcode is not smart enough to search headers by itself as we already linked the framework ?? lol 
@escamoteur if it works remember what I told you =>  one line fix :D 
thanks @bitsdojo !